medusa/name_parser/rules/rules.py

```diff
@@ -1220,8 +1222,8 @@ def when(self, matches, context):
                     to_remove.extend(matches.named('title', predicate=lambda match: match.value != values[0].value))
                     continue
 
-                logger.info(u"Guessed more than one '%s' for '%s': %s",
-                            name, matches.input_string, u','.join(unique_values), exc_info=False)
+                log.info(u"Guessed more than one '%s' for '%s': %s",
```

Comment:
Let's use {0} for Positional argument substitution for Medusa code okay? The we limit the amount of log patterns.